.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,50 @@
+name: Continuous Integration
+on:
+  push:
+    tags:
+    - '**'
+
+env:
+  GCLOUD_SERVICE_ACCOUNT_KEYFILE: ${{ secrets.GCLOUD_GH_ACTIONS_AUTH }}
+  GCLOUD_SERVICE_ACCOUNT_NAME: rasa-x-ci-github-actions@rasa-platform.iam.gserviceaccount.com
+
+jobs:
+  release:
+    name: Release new demo
+    runs-on: ubuntu-20.04
+    env:
+      TAG_NAME: ${GITHUB_REF#refs/tags/}
+    steps:
+      - name: Checkout git repository 🕝
+        uses: actions/checkout@v2
+
+      - uses: google-github-actions/setup-gcloud@daadedc81d5f9d3c06d2c92f49202a3cc2b919ba
+        name: Authenticate with gcloud 🎫
+        with:
+          version: "${{ env.GCLOUD_VERSION }}"
+          service_account_email: ${{ env.GCLOUD_SERVICE_ACCOUNT_NAME }}
+          service_account_key: ${{ secrets.GCLOUD_GH_ACTIONS_AUTH }}
+
+      - name: Authenticate docker 🎫
+        run: |
+          # Set up docker to authenticate via gcloud command-line tool.
+          gcloud auth configure-docker
+
+      - name: Build new image
+        run: |
+          RASA_SDK_VERSION=$(curl https://hub.docker.com/v2/repositories/rasa/rasa-sdk/tags |  jq -r '.results|.[]|.name' | grep -E "^[[:digit:]|\.|rc]+$" | sort -V | tail -n 1)
+          echo "Latest SDK version is ${RASA_SDK_VERSION}"
+          docker build \
+                -t gcr.io/rasa-platform/rasa-x-demo:${TAG_NAME} \
+                --build-arg RASA_SDK_VERSION=${RASA_SDK_VERSION} \
+                --cache-from gcr.io/rasa-platform/rasa-x-demo:${TAG_NAME} \
```

Comment:
@tczekajlo not sure we would ever have cache though, since I've removed all the caching parts :thinking: perhaps I can remove this argument as well? Dunno exactly how caching works for docker tbh

Reply:
Yes, you can get rid of it :)

Reply:
Done! :blush: